### DISCUSSION TITLE
Can I use JoyToKey and KeyToJoy at the same time ?

### DISCUSSION THREAD

**Discussion by AntoineXVI** [Key2Joy/Key2Joy#76] [Q&A] [answered]
Hello ! I'm a speedrunner and I want to configure buttons to keyboard and keyboard to buttons. So I DL your program in addition to JoyToKey.
I'm just trying to configure 1 mapping button, but the program deactivates the JoyToKey program: By moving the mouse, my controller is no longer detected but the mouse is detected as stick... Is there a solution?

**Comment by luttje** [comment, marked as answer]
Hi @AntoineXVI, thanks for your question!

I don't think what you are asking for is possible in most games. Most games only support one type of input and switch input-configuration when they detect it.

Additionally (from looking at the JoyToKey website) it seems that it too uses XInput, just like Key2Joy, so I imagine there will be conflicts.

Sadly, I'm not sure I can help with a solution at the moment.

Good luck!

**Reply by AntoineXVI** [reply]
Oh I see.
 Thanks for your fast reply !
